bodhi/server/validators.py

```diff
@@ -342,6 +355,18 @@ def validate_acls(request):
                                    "Database to check ACLs. Please "
                                    "try again later.")
                 return
+        elif acl_system == 'pagure':
+            try:
+                people, groups = package.get_pkg_committers_from_pagure(
+                    settings)
+                committers, watchers = people
+                groups, notify_groups = groups
+            except Exception, e:
+                log.exception(e)
+                error_msg = ('Unable to access Pagure to check ACLs. '
+                             'Please try again later.')
+                request.errors.add('body', 'builds', error_msg)
```

Comment:
I wonder if we want to give the user a clue about which type of error we got from Pagure? If it was a 404, should we tell the user, or would that definitely be a failure in the system somewhere?